### ISSUE TITLE
Bounding Box Rotating only on Y

### ISSUE THREAD

**Issue by lazarofl** [Microsoft/MRDesignLabs_Unity#44]
I have an issue where the bounding box target have Drag, AutoScale and Rotate X options checked.

I'm using a simple cube, the menu and bound boxes appear properly, but, when I change rotation Y to rotation X or Z it doesnt works. The appbar appears in both cases, but bounding boxes appear only if rotation Y is set and rotation X and Z are unset.

Is this an issue on MRDL or I need to configure something?

**InteractionManager** is set to **ManipulationTranslate**

**Comment by Railboy** [comment]
How are you doing the interaction? Are you seeing this on the device or in the Unity editor?

What axis flattening setting have you chosen on the Bounding Box Target?

**Comment by lazarofl** [comment]
Hi @Railboy.

I'm using Unity play mode during development, when I airptap cube it shows me appbar, but bounding boxes doesn't work.

flattening mode is set to **Flatten auto**.

![image](https://user-images.githubusercontent.com/913314/29297360-d3b8777e-8136-11e7-9589-317fd9414269.png)

My cube setup:

![image](https://user-images.githubusercontent.com/913314/29297383-f90f2fe0-8136-11e7-901d-b73da7eace68.png)

Collection is 2 meters in Z.

Behaviour is the same on Hololens.

I'm using unity 2017.1.0f3 and the latest version of MRDL, I updated my forked repo and rebuild the package to check.

I do a test now and when I set cube to rotate on X and flattering to **Flatten X** it works, but I need X, Y and Z.

![image](https://user-images.githubusercontent.com/913314/29297835-0b18e278-813a-11e7-9712-edb19a8ecab6.png)

Is this the spected behaviour?
